tests/test_git_browse.py

```diff
@@ -0,0 +1,231 @@
+import os, subprocess
+from testpath import MockCommand, modified_env
+
+github_origin = "https://github.com/vanpipy/git-extras"
```

Comment:
Is it possible to use `tj/git-extras` in the github test case?

Reply:
Update